etc/buildsys/fvconf.mk

```diff
@@ -117,14 +117,10 @@ ifeq ($(HAVE_OPENCV),1)
   VERSION_SPLITTED_OPENCV = $(call split,.,$(VERSION_OPENCV))
   VERSION_MAJOR_OPENCV    = $(word 1,$(VERSION_SPLITTED_OPENCV))
   VERSION_MINOR_OPENCV    = $(word 2,$(VERSION_SPLITTED_OPENCV))
-  ifeq ($(VERSION_MAJOR_OPENCV),4)
-    # We do not support opencv4 yet.
-    HAVE_OPENCV=0
-  endif
   CFLAGS_OPENCV      = -DHAVE_OPENCV -Wno-deprecated-declarations \
 		       $(shell $(PKGCONFIG) --cflags 'opencv$(OPENCV_VERSION_SUFFIX)') \
 		       $(CFLAG_W_NO_UNUSED_LOCAL_TYPEDEFS)
-  LDFLAGS_OPENCV     = $(shell $(PKGCONFIG) --libs-only-L 'opencv$(OPENCV_VERSION_SUFFIX)')
+  LDFLAGS_OPENCV     = $(shell $(PKGCONFIG) --libs-only-l 'opencv$(OPENCV_VERSION_SUFFIX)')
```

Comment:
If you do this then you should remove the complete block below as it adds redundant libraries to link against.

Reply:
I wanted to ask about this, because `--libs-only-l` does the same thing as `--libs` and just adds all the libraries to link against. Is that the behaviour that we want? 
`--libs-only-L` returns nothing under fedora 34 and opencv 4.5

Reply:
That's only true in this specific case. In general:
* `--libs` contains both `-l` and `-L` flags (both may be empty)
* `--libs-only-l` only contains `-l` flags, which are the libraries to link agains
* `--libs-only-L` contains additional directories where to look for the libaries. This is often empty, but does not need to be.

So you can either:
* use `--libs` to get all flags and remove the whole block below (that would actually be my preferred option)
* keep `-L` and the block below